### PR TITLE
statistics.md: update the content of analyze version2 OOM handling

### LINKED ISSUE #8136: analyze生成语句问题
File: [/master/statistics.md](https://docs.pingcap.com/zh/tidb/dev/statistics)
1，select distinct(concat('ANALYZE ',table_schema, '.', table_name,';')) from information_schema.tables, mysql.stats_histograms where stats_ver = 2 and table_id = tidb_table_id ;
不能运行，少了一个table关键字，应该是' ANALYZE TABLE '
2,如果用户表几千个，用鼠标来选择内容是不是有点麻烦。能把结果生成到一个文本方法放在这里么？

### PR DESCRIPTION
**Pull request by tiancaiamao**
### First-time contributors' checklist 

- [X] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Close #8136

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [X] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@tiancaiamao Could you please invite a technical review? Thanks!

**Review by winoros — APPROVED** [review]

**Comment by tiancaiamao** [comment]
PTAL @TomShawn

File: statistics.md [hunk 1]
```diff
@@ -26,6 +26,15 @@ TiDB 使用统计信息来决定[索引的选择](/choose-index.md)。变量 `ti
 >   ```sql
 >   select distinct(concat('DROP STATS ',table_schema, '.', table_name,';')) from information_schema.tables, mysql.stats_histograms where stats_ver = 2 and table_id = tidb_table_id ;
 >   ```
+>
+> - 如果上一条语句返回结果太长，不方便拷贝粘贴，可以将结果导出到临时文件后，再执行:
```
Comment (TomShawn) [inline statistics.md:30]: 「上一条语句」是指「如果 ANALYZE 语句是开启了自动 analyze 后 TiDB 自动执行的，请使用以下 SQL 语句生成 DROP STATS 的语句并执行」的语句吗？

Comment (tiancaiamao) [reply statistics.md:30]: 是指这条 SQL
```
select distinct(concat('DROP STATS ',table_schema, '.', table_name,';')) from information_schema.tables, mysql.stats_histograms where stats_ver = 2 and table_id = tidb_table_id
```

它有可能返回很长的结果，导致不方便拷贝内容

File: statistics.md [hunk 2]
```diff
@@ -26,6 +26,15 @@ TiDB 使用统计信息来决定[索引的选择](/choose-index.md)。变量 `ti
...
+>
+>   {{< copyable "sql" >}}
+>
```
Comment (TomShawn) [inline statistics.md:33]:
```suggestion
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**